api/nakadi-event-bus-api.yaml

```diff
@@ -1177,8 +1177,7 @@ definitions:
         allOf:
           - $ref: '#/definitions/EventTypeSchema'
         description: |
-          The schema for this EventType. This is expected to be a json schema in yaml format (other formats
-          might be added in the future).
```

Comment:
Yes, we have it there. We have the `type` field whose type is `array` of enums. The only valid value in the enum is `json_schema`.